roles/common/tasks/100_find_registry_ip.yml

```diff
@@ -0,0 +1,15 @@
+- set_fact:
+    re: "{{ ansible_default_ipv4.broadcast }}"
+    rs: "{{ ansible_default_ipv4.gateway }}"
+
+- name: Common | Checking network for free IP address for Virtual IP
```

Comment:
I would prefer to choose it automatically only when VIP is not set in group_vars. Otherwise, no one can control it.